digital_comms/ccam.py

```diff
@@ -144,7 +246,16 @@ def demand(self):
         return summed_demand / summed_area
 
     def coverage(self):
-        """Return proportion of population with capacity coverage over a threshold
+        """Calculate coverage as the proportion of population with capacity coverage over a threshold
+
+        Returns
+        -------
+        obj
+            Coverage in the local area district
+
+        Notes
+        -----
+        Function returns `0` when no postcode sectors are configured to the LAD.
```

Comment:
Are we missing the code for this method?

Reply:
No I don't think so. Line 261 - 262 implement this behavior.